exporters/writers/base_writer.py

```diff
@@ -65,6 +62,15 @@ def _get_compression_format(self):
                                      ''.format(FILE_COMPRESSION.keys()))
         return compression
 
+    def _get_write_buffer(self):
+        kwargs = {
+             'items_per_buffer_write': self.read_option('items_per_buffer_write'),
+             'size_per_buffer_write': self.read_option('size_per_buffer_write'),
+             'items_group_files_handler': self._items_group_files_handler(),
```

Comment:
or even better, the group files handler can be directly loaded by the writer, instead of being loaded with a method defined here